cmd/juju/application/deployer/bundlehandler_test.go

```diff
@@ -154,15 +154,15 @@ const wordpressBundle = `
 series: bionic
 applications:
   mysql:
-    charm: cs:mysql-42
+    charm: ch:mysql
```

Comment:
`charm: cs:mysql-42` in bundles is replaced by:
```
charm: cs:mysql
revision: 42
channel: stable
```

Not everywhere will be impacted, but in the case of charm upgrades etc via bundle deploy it matters.

Reply:
You mean `charm: ch:mysql \n...` (i.e. not `cs:mysql`), right?

Reply:
Do some, but not all